JSDemos/Demos/DataGrid/VirtualScrolling/description.md

```diff
@@ -1 +1,5 @@
-The **DataGrid** widget supports virtual record scrolling, allowing end users to&nbsp;navigate the contents of&nbsp;the widget by&nbsp;using its vertical scrollbar. In&nbsp;this demo, the **DataGrid** is&nbsp;connected to&nbsp;100,000 records and virtual scroll mode has been enabled to&nbsp;simplify record navigation and eliminate the use of&nbsp;data pagers.
\ No newline at end of file
+If the **DataGrid** component works with a large dataset, you can enable virtual scrolling to optimize data loading and navigation. The component calculates the overall number of visible rows and displays a scrollbar so users can jump to any location. When users release the scroll thumb, the control loads the records to be displayed in the viewport and removes other rows from memory. 
```

Comment:
I'd like to make this more specific than "works with" and "jump to any location". Could we, for example, say:

```suggestion
If the **DataGrid** component is bound to a large dataset, you can enable the virtual scroll feature to optimize data load times and improve user navigation. The component calculates the overall number of visible rows and displays a scrollbar that allows users to navigate to any section of rows. When users release the scroll thumb, the control loads records to be displayed in the viewport and removes other rows from memory. 
```